core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -517,6 +517,9 @@ var TopicEditorPage = function() {
   };
 
   this.saveTopic = async function(commitMessage) {
+    await waitFor.visibilityOf(
+      saveTopicButton,
+      'Save topic button takes too long to appear');
```

Comment:
This works, but at this point you might as well just use `action.click`

Reply:
done